Customizing/global/plugins/Services/Cron/CronHook/ReportMaster/classes/ReportBase/class.catSelectableReportTableGUI.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+class catSelectableReportTableGUI extends catTableGUI {
```

Comment:
In dieser Klasse hätten mir Kommentare sehr geholfen, verstehe kaum, was das macht.